packages/core/src/api/guild.ts

```diff
@@ -0,0 +1,735 @@
+	 * @param templateCode - The code of the template to delete
+	 */
+	public async deleteTemplate(guildId: string, templateCode: string) {
+		await this.rest.delete(Routes.guildTemplate(guildId, templateCode));
+	}
+
+	/**
+	 * Fetches all the stickers for a guild
+	 *
+	 * @param guildId - The id of the guild to fetch the stickers for
+	 */
+	public async getStickers(guildId: string) {
+		return (await this.rest.get(Routes.guildStickers(guildId))) as APISticker[];
+	}
+
+	/**
+	 * Fetches a sticker for a guild
+	 *
+	 * @param guildId - The id of the guild to fetch the sticker for
+	 * @param stickerId - The id of the sticker to fetch
+	 */
+	public async getSticker(guildId: string, stickerId: string) {
+		return (await this.rest.get(Routes.guildSticker(guildId, stickerId))) as APISticker;
+	}
+
+	/**
+	 * Edits a sticker for a guild
+	 *
+	 * @param guildId - The id of the guild to edit the sticker for
+	 * @param stickerId - The id of the sticker to edit
+	 * @param options - The options for editing the sticker
+	 * @param reason - The reason for editing the sticker
+	 */
+	public async editSticker(
+		guildId: string,
+		stickerId: string,
+		options: RESTPatchAPIGuildStickerJSONBody,
+		reason?: string,
+	) {
+		return (await this.rest.patch(Routes.guildSticker(guildId, stickerId), {
+			reason,
+			body: options,
+		})) as APISticker;
+	}
+
+	/**
+	 * Deletes a sticker for a guild
+	 *
+	 * @param guildId - The id of the guild to delete the sticker for
+	 * @param stickerId - The id of the sticker to delete
+	 * @param reason - The reason for deleting the sticker
+	 */
+	public async deleteSticker(guildId: string, stickerId: string, reason?: string) {
+		await this.rest.delete(Routes.guildSticker(guildId, stickerId), { reason });
+	}
+
+	/**
+	 * Fetches the audit logs for a guild
+	 *
+	 * @param guildId - The id of the guild to fetch the audit logs for
+	 * @param options - The options for fetching the audit logs
+	 */
+	public async getAuditLogs(guildId: string, options: RESTGetAPIAuditLogQuery = {}) {
+		return (await this.rest.get(Routes.guildAuditLog(guildId), {
+			query: makeURLSearchParams(options as Record<string, unknown>),
+		})) as APIAuditLog[];
+	}
+
+	/**
+	 * Fetches all auto moderation rules for a guild
+	 *
+	 * @param guildId - The id of the guild to fetch the auto moderation rules for
+	 */
+	public async getAutoModRules(guildId: string) {
+		return (await this.rest.get(Routes.guildAutoModerationRules(guildId) as `/${string}`)) as APIAutoModerationRule[];
+	}
+
+	/**
+	 * Fetches an auto moderation rule for a guild
+	 *
+	 * @param guildId - The id of the guild to fetch the auto moderation rule from
+	 * @param ruleId - The id of the auto moderation rule to fetch
+	 */
+	public async getAutoModRule(guildId: string, ruleId: string) {
+		return (await this.rest.get(
+			Routes.guildAutoModerationRule(guildId, ruleId) as `/${string}`,
+		)) as APIAutoModerationRule;
+	}
+}
```

Comment:
The [documentation](https://discord.com/developers/docs/resources/auto-moderation) does not use "Auto Mod" or any other shortened variant, so I believe we should tweak the naming here.

```suggestion
	/**
	 * Fetches all auto moderation rules for a guild
	 *
	 * @param guildId - The id of the guild to fetch the auto moderation rules for
	 */
	public async getAutoModerationRules(guildId: string) {
		return (await this.rest.get(Routes.guildAutoModerationRules(guildId) as `/${string}`)) as APIAutoModerationRule[];
	}

	/**
	 * Fetches an auto moderation rule for a guild
	 *
	 * @param guildId - The id of the guild to fetch the auto moderation rule from
	 * @param ruleId - The id of the auto moderation rule to fetch
	 */
	public async getAutoModerationRule(guildId: string, ruleId: string) {
		return (await this.rest.get(
			Routes.guildAutoModerationRule(guildId, ruleId) as `/${string}`,
		)) as APIAutoModerationRule;
	}
}
```